camunda-formio-tasklist-vue/src/services/CamundaRest.ts

```diff
@@ -279,4 +279,15 @@ export const CamundaRest = {
       `/${engine}/process-definition/${processDefinitionId}/xml`
     );
   },
+
+  async getProcesActivity (
+    bearerToken: any,
+    processInstanceId: string,
+    CamundaUrl: any
```

Comment:
CamundaUrl is also string.

In case of any other cases this is not followed. You can update there as well.